fellow-submissions/Cielo-Raymundo.md

```diff
@@ -0,0 +1,138 @@
+# Problem Set 2.4 - Scope
+
+**1. How do variables declared with `let`, `const`, and `var` differ? Be sure to touch on _scope_, _reassignment_, and _hoisting_ for each.**
+    Variables starting with var can have either global scope or a function scope, which is like having a bubble around the variable
```

Comment:
instead of variables _starting_ with `var`... say variables _**declared**_ with `var`...